cluster/src/test/scala/org/apache/spark/sql/NorthWindTest.scala

```diff
@@ -57,6 +57,30 @@ class NorthWindTest
     validatePartitionedColumnTableQueries(snc)
   }
 
+  // enable if transformations are supported in plan-caching.
+  ignore("SNAP-2451"){
+    createAndLoadColumnTables(snc)
```

Comment:
I think this test should be enabled. Do you see failure in this?

Reply:
This test is failing when planCaching is enabled.